### PR TITLE
task/WG-285: update potree converter

### PR DESCRIPTION
**Pull request by nathanfranklin**
## Overview: ##

This PR bumps Potree Converter.  

This PR does the following:
* reworks Dockerfile for workers
    * uses Ubuntu as the base image (instead of an older PDAL image)
    * installs conda and installs PDAL via conda 
* bumps Potree Converter version

## Related Jira tickets: ##

* [WG-285](https://tacc-main.atlassian.net/browse/WG-285) - issue with loading Oso_MiniRanger_ULS_20240315.laz

## Testing Steps: ##
1. Build the updated image: `make build`
2. Run hazmapper and import a laz point cloud and confirm that it converted and the viewer works file
3. If you want to test with Oso_MiniRanger_ULS_20240315.laz, a copy is in https://designsafe-ci.org/data/browser/projects/PRJ-4576

## Notes: ##

TODO
- [ ] We pinged RAPID for additional testing on Oct 4th as this bumps the potree viewer from 1.6 to 1.8 and curious if everything works as expected for their use cases. It is deployed to staging for their testing.
- [x] update deployed nginx conf
- [x] confirm which version of potreeconvert (going with c2328c4 which is v2.1.1 with some additional fixes)
- [x] confirm if we need LAStools still for building PotreeConverter

### CODE REVIEW

File: devops/Dockerfile.worker [hunk 1]
```diff
@@ -1,46 +1,92 @@
...
 RUN poetry install
 
-ENV PYTHONPATH "${PYTHONPATH}:/app"
-
-WORKDIR /
+# Populate image with geoapi and set PYTHONPATH
 RUN mkdir app
 COPY geoapi /app/geoapi
 WORKDIR /app/geoapi
+ENV PYTHONPATH "${PYTHONPATH}:/app"
```
Comment (taoteg) [inline devops/Dockerfile.worker:70]: This throws a docker warning for using the legacy variable format, instead try: 
`ENV PYTHONPATH="${PYTHONPATH}:/app"`

Comment (taoteg) [reply devops/Dockerfile.worker:70]: Same tweak goes for line 22 of the main Dockerfile (not the workers one), but its not in this PR...

Comment (nathanfranklin) [reply devops/Dockerfile.worker:70]: thanks.

**Review by sophia-massie — APPROVED** [review]
LGTM tested locally. Thanks for the changes

**Pull request merged**